pegen/pegen.c

```diff
@@ -875,24 +849,15 @@ _get_exprs(Parser *p, asdl_seq *seq)
     return new_seq;
 }
 
-/* Wrapper for _Py_Compare, so that the call in the grammar stays concise */
-expr_ty
-Pegen_Compare(Parser *p, expr_ty expr, asdl_seq *pairs, int lineno, int col_offset,
```

Comment:
We should remove this from the header file and the helpers.md file now that it's not used anymore.